### ISSUE TITLE
Error in vgg_atrous.py when BatchNorm is enabled

### ISSUE THREAD

**Issue by Wallart** [dmlc/gluon-cv#97]
https://github.com/dmlc/gluon-cv/blob/af5ff238b3b9ea5073c80b54619b1331578c79da/gluoncv/model_zoo/ssd/vgg_atrous.py#L121

There is a small mistake :  "stage.add(nn.BatchNorm())" should be "extra.add(nn.BatchNorm())"

**Comment by zhreshold** [comment]
This has already fixed in master. If you are using pip, you can reinstall using `pip install gluoncv --pre`
